Ex04/index.js

```diff
@@ -0,0 +1,76 @@
+const express = require('express')
+    //const bodyParser = require('body-parser');
+const fs = require('fs');
+let app = express()
+const port = 8080
+
+app.get('/tasks', (req, res) => { //show all tasks in tasks.json   
+    console.log('GET /tasks');
+
+    fs.readFile('./tasks.json', (err, content) => {
+        if (err) {
+            console.error(err);
+            return
+        }
+        res.send(JSON.parse(content));
+    });
+});
+
+
+app.post('/tasks/new', (req, res) => { //add task to tasks.json   
+    console.log('POST /tasks/new');
+
+    //get from the URL
+    let taskId = req.query.id;
+    let theTask = req.query.task;
+
+    let updatedTasks;
+
+    fs.readFile('./tasks.json', (err, content) => {
+        if (err) {
+            console.error(err);
+            return
```

Comment:
In a case of error you should let the user know about it, it is better then returning an empty page